MekHQ/src/mekhq/gui/dialog/ResolveScenarioWizardDialog.java

```diff
@@ -529,8 +516,15 @@ private void initComponents() {
             prisonerKiaBtns.add(kiaCheck);
             kiaCheck.setSelected(status.getHits() > 5 || status.isDead());
             gridBagConstraints.gridx = 3;
-            gridBagConstraints.weightx = 1.0;
             pnlPrisonerStatus.add(kiaCheck, gridBagConstraints);
+            btnViewPrisoner = new JButton("View Personnel");
+            btnViewPrisoner.setEnabled(!status.isDead());
+            btnViewPrisoner.setActionCommand(status.getId().toString());
+            btnViewPrisoner.addActionListener(new ViewPrisonerListener());
```

Comment:
This is fine, but for future reference there's syntax that allows you to declare an in-line handler so you don't have to create a new class.